src/compile/data/summary.ts

```diff
@@ -45,8 +46,11 @@ export namespace summary {
           meas['*']['count'] = true;
           /* tslint:enable:no-string-literal */
         } else {
-          meas[fieldDef.field] = meas[fieldDef.field] || {};
-          meas[fieldDef.field][fieldDef.aggregate] = true;
+          const field = fieldDef.field;
+          if (isString(field)) {
```

Comment:
`// FIXME  support iterator`

I guess we have to add another parameter as `iteratorMap` so we can replace repeatRef with current value of the iterator.